katharsis-core/src/main/java/io/katharsis/errorhandling/exception/InternalServerErrorException.java

```diff
@@ -0,0 +1,22 @@
+package io.katharsis.errorhandling.exception;
+
+import io.katharsis.errorhandling.ErrorData;
+import io.katharsis.response.HttpStatus;
+
+public class InternalServerErrorException extends KatharsisMappableException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This class has 6 parents which is greater than 5 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)